src/Controllers/MaterialCommentController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Uasoft\Badaso\Module\LMSModule\Controllers;
+
+use Exception;
+use Illuminate\Http\Request;
+use Illuminate\Validation\ValidationException;
+use Uasoft\Badaso\Controllers\Controller;
+use Uasoft\Badaso\Helpers\ApiResponse;
+use Uasoft\Badaso\Module\LMSModule\Models\LessonMaterial;
+use Uasoft\Badaso\Module\LMSModule\Models\MaterialComment;
+
+class MaterialCommentController extends Controller
+{
+    public function add(Request $request)
+    {
+        try {
+            $user = auth()->user();
+            $request->validate([
+                'material_id' => 'required|integer',
+                'content' => 'required|string|max:65535',
+            ]);
+
+            $announcement = LessonMaterial::where('id', $request->input('material_id'))
+                ->first();
+
+            if (! $announcement) {
```

Comment:
and I think there should be a validation where user that is not in the course cannot add a comment. Perhaps using:
```php
            if (! CourseUserHelper::isUserInCourse(
                $user->id,
                $lessonMaterial?->course_id,
            ))
```